File: Algorithms/Warmup/Solve me first/solve_me_first.c
#include <stdio.h>

int sum(int a, int b);

int main(){
  int num1, num2;
  scanf("%d\n%d", &num1, &num2);
  printf("%d", sum(num1, num2));
  return 0;
}

int sum(int a, int b){
  return a + b;
}

File: Algorithms/Warmup/Solve me first/SolveMeFirst.java
import java.util.Scanner;

public class SolveMeFirst{
  static int sum(int a, int b){
    return a + b;
  }

  public static void main(String [] args){
    int num1, num2;
    Scanner in = new Scanner(System.in);
    num1 = in.nextInt();
    num2 = in.nextInt();
    System.out.println(sum(num1, num2));
  }
}
